src/registrar/models/domain_request.py

```diff
@@ -1332,3 +1332,72 @@ def _form_complete(self, request):
         if not is_complete or not self._is_general_form_complete(request):
             return False
         return True
+
+    """the following converted_ property methods get their respective field names from portfolio,
```

Comment:
Nitpick: We aren't really getting "field names", we're getting data.  Perhaps reword to something like "The following converted_ property methods get field data from this domain request's portfolio, if there is an associated portfolio.  If not, they return data from the domain request model.

Reply:
Okay will update!